verifier/src/main/kotlin/tech/figure/classification/asset/verifier/config/EventStreamProvider.kt

```diff
@@ -0,0 +1,16 @@
+package tech.figure.classification.asset.verifier.config
+
+import io.provenance.eventstream.stream.clients.BlockData
+import tech.figure.classification.asset.verifier.provenance.AssetClassificationEvent
+
+interface EventStreamProvider {
+    suspend fun currentHeight(): Long?
+    suspend fun processBlockForHeight(
```

Comment:
This should probably be renamed to something more accurate to its usage, since it essentially starts processing all blocks continually from the given height.  Something like `startProcessingFromHeight` maybe?  The name right now makes it sound like it'll process just a single block

Reply:
fair enough, updated to `startProcessingFromHeight`